cxf/3.0.5.wso2v1/pom.xml

```diff
@@ -169,6 +169,12 @@
             <version>3.2.9.wso2v1</version>
             <optional>true</optional>
         </dependency>
```

Comment:
AFAIK this is already released bundle. Therefore you have to create a new bundle